client/state/action-types.js

```diff
@@ -284,6 +284,8 @@ export const FOLLOWERS_REQUEST_ERROR = 'FOLLOWERS_REQUEST_ERROR';
 export const FOLLOWER_REMOVE_ERROR = 'FOLLOWER_REMOVE_ERROR';
 export const FOLLOWER_REMOVE_REQUEST = 'FOLLOWER_REMOVE_REQUEST';
 export const FOLLOWER_REMOVE_SUCCESS = 'FOLLOWER_REMOVE_SUCCESS';
+export const FULL_HEIGHT_LAYOUT_ON = 'FULL_HEIGHT_LAYOUT_ON';
+export const FULL_HEIGHT_LAYOUT_OFF = 'FULL_HEIGHT_LAYOUT_OFF';
```

Comment:
There can be just one action, `FULL_HEIGHT_LAYOUT_TOGGLE`. No need to spam the already long list of Redux actions 🙂